src/components/Author.js

```diff
@@ -8,7 +8,9 @@ import twitterLogo from '../../content/assets/twitter-logo.svg'
 
 const Author = ({ author, ...props }) => (
   <Div column itemsCenter listTop={16} {...props}>
-    <Avatar src={author.avatar.publicURL} alt={author.name} />
+    <Link to={`/authors/${author.id}`}>
+      <Avatar src={author.avatar.publicURL} alt={author.name} />
```

Comment:
I think I'd wrap the link around the `{author.name}` as well.

Reply:
👍